src/Columns/ColumnNullable.h

```diff
@@ -121,6 +121,7 @@ class ColumnNullable final : public COWHelper<IColumn, ColumnNullable>
     const IColumn & getNestedColumn() const { return *nested_column; }
 
     const ColumnPtr & getNestedColumnPtr() const { return nested_column; }
+    ColumnPtr getNestedColumnCopy() const;
```

Comment:
Comment is needed.
It is not obvious why it is not just `IColumn::mutate()` or `cloneResized` (because we replace nulls to zeroes).
And (why) it works only for numbers.